src/lesson10/exceprions/userValidationService/ValidationUserService.java

```diff
@@ -0,0 +1,30 @@
+package lesson10.exceprions.userValidationService;
+
+public class ValidationUserService {
+
+    private void validateUserFirstName(User user) {
+        int lengthUserFirstName = user.getFirstName().length();
```

Comment:
You're not validating if firstName is null

Reply:
Since a string is an object, should it be checked for null?

Reply:
Yes of course, in java any object can be null and you will get `NullPointerException`